### DISCUSSION TITLE
Recommended way to disable this package during testing?

### DISCUSSION THREAD

**Discussion by MrMage** [spatie/laravel-google-fonts#47] [Q&A]
I have `Http::preventStrayRequests();` enabled in my tests and am running `php artisan google-fonts:fetch` before running tests in CI to ensure that this package doesn't cause test failures due to trying to fetch fonts during test runtime.

However, over the course of the past few weeks, this command often fails with:

```
In GoogleFonts.php line 130:
                         
  Undefined array key 1
```

This then also causes my tests to fail:

```
  Expected response status code [200] but received 500.
Failed asserting that 500 is identical to 200.

The following exception occurred during the last request:

RuntimeException: Attempted request to [https://fonts.googleapis.com/css?family=Nunito+Sans:300,400,400i,600,700] without a matching fake. in /var/www/html/vendor/laravel/framework/src/Illuminate/Http/Client/PendingRequest.php:1249
Stack trace:
```

Is there an easy way to have this package do nothing during tests, so that the tests become more hermetic?
Or should I just wrap the `@googlefonts` directive in a call to `@if(!env('testing'))`?

**Comment by sebastiandedeyne** [comment]
I wouldn't mind an `enabled` configuration option we could tie to the app environment. But for now, an environment check in your views sounds good.

**Comment by clementmas** [comment]
It would be great to add an option to turn it off.

Meanwhile I had to update all my layouts to use:

```php
@if (!app()->environment('testing'))
    @googlefonts
@endif
```
